tasks/push-snapshot/README.md

```diff
@@ -4,11 +4,18 @@ Tekton task to push snapshot images to an image registry using `cosign copy`.
 
 ## Parameters
 
-| Name         | Description                                                               | Optional | Default value        |
-|--------------|---------------------------------------------------------------------------|----------|----------------------|
-| snapshotPath | Path to the JSON string of the mapped Snapshot spec in the data workspace | Yes      | mapped_snapshot.json |
-| dataPath     | Path to the JSON string of the merged data to use in the data workspace   | Yes      | data.json            |
-| retries      | Retry copy N times                                                        | Yes      | 0                    |
+| Name               | Description                                                               | Optional | Default value        |
+|--------------------|---------------------------------------------------------------------------|----------|----------------------|
+| snapshotPath       | Path to the JSON string of the mapped Snapshot spec in the data workspace | Yes      | mapped_snapshot.json |
+| dataPath           | Path to the JSON string of the merged data to use in the data workspace   | Yes      | data.json            |
+| pushSourceContainer| Push the source container to the mapped target                            | Yes      | "false"              |
```

Comment:
Nit: For consistency, I'd use `false` instead of `"false"`. All tekton params are strings anyway, so no need to signify that it's a string and not a bool.

Reply:
Noted and corrected. Thanks!